csharp/src/Ice/Proxy.cs

```diff
@@ -946,12 +925,9 @@ public string ice_getFacet()
     /// <returns>The proxy with the new facet.</returns>
     public ObjectPrx ice_facet(string newFacet)
     {
-        if (newFacet == null)
-        {
-            newFacet = "";
-        }
+        newFacet ??= "";
```

Comment:
If we're handling the case of a null string here, shouldn't the parameter be
`string? newFacet`?

Reply:
The parameter should be non-nullable. We just provide this null handling for backwards compatibility with existing nullability oblivious code that used null as a synonym for "".